tests/integration/dual-channel-replication.tcl

```diff
@@ -815,6 +815,37 @@ start_server {tags {"dual-channel-replication external:skip"}} {
         } else {
             fail "Primary should abort sync"
         }
+        stop_write_load $load_handle0
+        stop_write_load $load_handle1
+        stop_write_load $load_handle2
+    }
+}
+
+start_server {tags {"dual-channel-replication external:skip"}} {
```

Comment:
Can `restart_instance` be used instead?

Reply:
`restart_instance` could be used here, but splitting tests to different servers simplifies debugging by separating logs and avoids unnecessary test coupling. These tests don't require shared server state.

Reply:
I'm wondering if there's a before-test hook to always restart the servers and cleanup the logs.